examples/README.md

```diff
@@ -0,0 +1,94 @@
+# Examples
+
+This package contains all of the sources that appear in the Language Tour and the Library Tour, including the smallest of code excerpts.
+
```

Comment:
I think you'll want to add a writer-oriented manual. Something like

1. Add sample code as `.dart` files to `examples/directory/`
2. Annotate it with `#docregion`
3. Add the same code as excerpts to your `.md` articles
4. Annotate it with `<annotation>`

Reply:
Switching to a writer-oriented perspective is an excellent idea.

Reply:
I'll followthrough via #436.